main/preload.js

```diff
@@ -52,5 +52,19 @@ contextBridge.exposeInMainWorld('electron', {
     setFilAddress: (/** @type {string | undefined} */ address) => ipcRenderer.invoke('station:setFilAddress', address),
     getOnboardingCompleted: () => ipcRenderer.invoke('station:getOnboardingCompleted'),
     setOnboardingCompleted: () => ipcRenderer.invoke('station:setOnboardingCompleted')
+  },
+  stationEvents: {
+    onActivityLogged: (/** @type {() => Function} */ callback) => {
```

Comment:
should we make the `Function` type more specific?

Reply:
[e31bfe8](https://github.com/filecoin-project/filecoin-station/pull/144/commits/e31bfe884190e67c2ad3073b71126d5f64332567)